com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Output.Drawers.cs

```diff
@@ -25,23 +25,12 @@ partial class Output
 #endif
                     Drawer_Output_RenderTarget,
                     Drawer_Output_MSAA_Warning,
-                    Drawer_Output_AllowDynamicResolution,
+                    CameraUI.Output.Drawer_Output_AllowDynamicResolution,
                     Drawer_Output_Depth,
-                    Drawer_Output_NormalizedViewPort
+                    CameraUI.Output.Drawer_Output_NormalizedViewPort
                 )
             );
 
-            static void Drawer_Output_AllowDynamicResolution(SerializedHDCamera p, Editor owner)
-            {
-                EditorGUILayout.PropertyField(p.allowDynamicResolution, Styles.allowDynamicResolution);
-                p.baseCameraSettings.allowDynamicResolution.boolValue = p.allowDynamicResolution.boolValue;
-            }
-
-            static void Drawer_Output_NormalizedViewPort(SerializedHDCamera p, Editor owner)
-            {
-                EditorGUILayout.PropertyField(p.baseCameraSettings.normalizedViewPortRect, Styles.viewport);
-            }
-
             static void Drawer_Output_Depth(SerializedHDCamera p, Editor owner)
             {
                 EditorGUILayout.PropertyField(p.baseCameraSettings.depth, Styles.depth);
```

Comment:
Based on this PR it's not clear to me what's the logic for keeping some UI functions in HD and moving others to Core. For example, this function also refers to `baseCameraSettings` which I guess is "common functionality" between SRPs? Could you explain the logic a bit - do you have planned what the URP UI will have and you have only moved the things URP will need?

Reply:
I checked what URP was using from the engine source code that was replicated on HDRP and I moved to Core.

But you are right that there is not a hard conection with the `HDAdditionalCameraData`, so I will move this two drawers to core.